02.calendar/cal.rb

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env ruby
+require 'optparse'
+require 'date'
+options = ARGV.getopts('y:','m:')
+year = options["y"].to_i
+month = options["m"].to_i
+
+# エラー入力の際に今日の日付を表示する設定
+begin
+date = Date.new(year, month, 1)
+rescue
+year = Date.today.year
+month = Date.today.month
+date_today = Date.today.mday
+date = Date.new(year, month, 1)
+end
+
+# 月と年の表示/曜日の表示
+puts date.strftime("%-m月%Y").center(20)
+puts "日 月 火 水 木 金 土"
+
+# 月によって異なる第１行のスペースを文字列にして配列に追加
+day = date.strftime('%a')
+monthly_date=[]
+case day
+    when "Mon"
+    monthly_date.push("   ")
+    when "Tue"
+    2.times{monthly_date.push("   ")}
```

Comment:
後々rubocopを使えば機械的に指摘してもらえることではありますが、スペースの空け方を以下のようにした方が良いです。他のブロックを使っている箇所も同様です。
```suggestion
    2.times { monthly_date.push("   ") }
```
 参考：https://docs.ruby-lang.org/ja/latest/method/Integer/i/times.html

Reply:
修正いたしました。